build/policy/files.rego

```diff
@@ -9,10 +9,11 @@ package files
 
 import future.keywords.in
 
-import data.helpers.endswith_any
-import data.helpers.last_indexof
+import data.helpers.extension
+import data.helpers.directory
 
 filenames := {f.filename | some f in input}
+logo_exts := {"png", "svg"}
```

Comment:
We should update the docs to reflect this:
https://github.com/open-policy-agent/opa/blob/35184350b5380bfa63bd63688ee3a24d07330856/docs/README.md?plain=1#L334